frontend/src/stores/useRecentScansStore.jsx

```diff
@@ -0,0 +1,39 @@
+import { create } from "zustand";
+import axios from "axios";
+
+import { JOB_RECENT_SCANS, JOB_RECENT_SCANS_USER } from "../constants/api";
+
+const useRecentScansStore = create((set, _get) => ({
+  loadingScansUser: true,
+  loadingScans: true,
```

Comment:
Silly requests: I suggest to rename this and the other field to `loadingScansInsertedAnalyzable` or something like this